src/ReverseProxy.TelemetryConsumption/TelemetryConsumptionExtensions.cs

```diff
@@ -38,6 +39,12 @@ public static IServiceCollection AddTelemetryConsumer(this IServiceCollection se
         {
             var implementsAny = false;
 
+            if (consumer is IWebSocketsTelemetryConsumer)
+            {
+                services.TryAddEnumerable(new ServiceDescriptor(typeof(IWebSocketsTelemetryConsumer), consumer));
```

Comment:
This is a matter of style. How about this one?
```suggestion
                services.TryAddEnumerable(ServiceDescriptor.Singleton<IWebSocketsTelemetryConsumer>(consumer);
```

Reply:
Since `consumer` in this case is `object`, `.Singleton<T>(T foo)` can't be used.

But it could be changed to
```c#
if (consumer is IWebSocketsTelemetryConsumer webSocketsTelemetryConsumer)
{
    services.TryAddEnumerable(ServiceDescriptor.Singleton(webSocketsTelemetryConsumer));
}
```

which does still seem nicer since we're avoiding the `typeof` 👍.